cla_public/apps/checker/fields.py

```diff
@@ -150,7 +154,9 @@ def process_data(self, value):
 class MoneyIntervalForm(NoCsrfForm):
     """Money amount and interval subform"""
     per_interval_value = MoneyField(validators=[Optional()])
-    interval_period = SelectField('', choices=MONEY_INTERVALS)
+    interval_period = SelectField('',
+                                  choices=MONEY_INTERVALS,
+                                  coerce=coerce_unicode_if_value)
```

Comment:
I know this is PEP8 compliant, but I don't like indentation to an arbitrary depth like this. It's much better IMO like this:
```python
interval_period = SelectField(
    '',
    choices=MONEY_INTERVALS,
    coerce=coerce_unicode_if_value)
```